spec/index.bs

```diff
@@ -205,14 +205,17 @@ it's not easy to distinguish one from the other.
 This document organizes the injection sinks into groups, based on the
 capabilities that sinks in a given group have. [=Enforcement=] for groups is controlled via <a>trusted-types-sink-group</a> values.
 
-### HTML injection sinks ### {#html-injection-sinks}
```

Comment:
Perhaps it might be easier to just merge those two existing sections, and separate them once there is another use case specified?

https://github.com/w3c/trusted-types/pull/407

Reply:
Agree.